plugin/trino-kafka/src/main/java/io/trino/plugin/kafka/KafkaAdminFactory.java

```diff
@@ -14,17 +14,18 @@
 
```

Comment:
cmt msg: What about consumers (I see changes in code but mention in cmt msg) and schema registry (I don't see anything anywhere)?

Reply:
What about schema registry?

Reply:
I guess for SchemaRegistry it would be a bigger change. Currently we don't have a Factory like method for SchemaRegistry.

Reply:
Ok, lets put it as a out of the scope of this PR then.